Tests/Benchmarks/SolenoidFieldBenchmark.cpp

```diff
@@ -63,13 +65,30 @@ int main(int argc, char* argv[]) {
     return {r * std::cos(phi), r * std::sin(phi), z};
   };
 
+  auto csv = [&](const std::string& name, auto res) {
+    auto& os = std::cerr;
+
+    os << name << "," << res.run_timings.size() << "," << res.iters_per_run
+       << "," << res.totalTime().count() << "," << res.runTimeMedian().count()
+       << "," << 1.96 * res.runTimeError().count() << ","
+       << res.iterTimeAverage().count() << ","
+       << 1.96 * res.iterTimeError().count();
+
+    os << std::endl;
+  };
+
+  std::cerr << "name,runs,iters,total_time,run_time_median,run_time_error,iter_"
+               "time_average,iter_time_error"
+            << std::endl;
```

Comment:
Could you write to a file instead of hijacking stderr for this?

Reply:
Sure. I personally like having it in this way because then you **can** just read it on the prompt, but I can change it anyway.

Reply:
Well, there is already a fine tune human-readable display of the results on stdout, so I'm not sure why you would want to mentally parse CSV instead...

Reply:
For rediretion. But you're right, I'll change this.

Reply:
Ok. Should go to a (hard coded) output file now.